src/main/java/org/libtorrent4j/AnnounceFlags.java

```diff
@@ -0,0 +1,27 @@
+package org.libtorrent4j;
+
+/**
+ * @author aldenml
+ */
+public final class AnnounceFlags {
```

Comment:
Codacy found an issue: [The utility class name 'AnnounceFlags' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5839370)